### PR TITLE
Fixed the issue with default index pages have broken image links

### PR DESCRIPTION
**Pull request by sujaydeymountblue**
#### Description of what you did:
Fixed the issue when the main website got swapped over and three people's image are missing

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello! When I setup the mono repo, I'm not able to access your new files, they are not displayed. 
Screen Shot 2020-06-02 at 3 24 12 PM
In the documentation you JSON object you updated, the indentation is not correct.

**Comment by sujaydeymountblue** [comment]
I will rectify the issue in some time and send the PR @lauriejim

File: docs/3.0.0-beta.x/concepts/configurations.md [hunk 1]
```diff
@@ -302,7 +302,14 @@ You can find [supported database and versions](../installation/cli.html#database
         "password": "root",
         "database": "strapi"
       },
-      "options": {}
+      "options": {
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/concepts/configurations.md:305]: Can you revert those changes

File: docs/v3.x/plugin-development/plugin-architecture.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ plugin/
 |               └─── fr.json
 └─── config/ # Contains the configurations of the plugin
 |     └─── functions/
-|     |    └─── bootstrap.js # Asynchronous bootstrap function that runs before the app gets started
+|     |    └───  # Asynchronous bootstrap function that runs before the app gets started
```
Comment (alexandrebodin) [inline docs/v3.x/plugin-development/plugin-architecture.md:24]: Can you revert those changes

**Comment by alexandrebodin** [comment]
I'm going to close this issue for inactivy. Feel free to reopen when you get some time to work on it :)

**Pull request closed without merging**